pkg/backporter/backporter.go

```diff
@@ -184,6 +197,32 @@ const clonesTemplateConstructor = `
     </form>
 </div>`
 
+const helpTemplateConstructor = `
+<h2 id="title"><a href="#title">Why a bugzilla-backporter?</a></h2>
+
+<p>
+The <code>Bugzilla Backporter</code> would allow the user to easily understand how different clones
```

Comment:
Call the thing consistently either "Bugzilla Backporter" or "bugzilla-backporter" (I think for humans we prefer the former).

Reply:
Also I don't think we need the conditional - let's just say "backporter allows" instead of "would allow" (in the whole text).

Reply:
Yes, I will change that